libbeat/common/mapstr.go

```diff
@@ -138,6 +138,15 @@ func (m MapStr) Put(key string, value interface{}) (interface{}, error) {
 	return walkMap(key, m, mapStrOperation{putOperation{value}, true})
 }
 
+// SetValue dereferences a value if it is a pointer. Sets the dereferenced value if not nil.
```

Comment:
@simitt No worries about reverting the commit. Just remove the code again in the follow up commit. I think so far we never reverted a commit, we only move forward :-D